src/components/FormComponents/InputController/InputController.tsx

```diff
@@ -42,19 +48,19 @@ export const InputController = <T extends FieldValues>({
               <StyledTextField
                 {...textFieldProps}
                 onChange={onChange}
-                value={textFieldProps.type === 'number' && value < 1 ? 1 : value}
+                value={
+                  textFieldProps.type === 'number' && value < defaultNumberValue
```

Comment:
 Maybe it makes sense to add types  to the enum?
 textFieldProps.type === FieldType.Number